MusicAlbum/musicalbum.rb

```diff
@@ -0,0 +1,18 @@
+include '../items'
+include '../Genre/genre'
+
+class MusicAlbum < Genre
```

Comment:
Inherit Music Album from Item class not from Genre

Reply:
yes im going to change it